Masinette add kitemaker pagertree (#9747)
getsentry/sentry-docs: docs/product/integrations/notification-incidents/kitemaker/index.mdx

---
title: Kitemaker
sidebar_order: 1

description: "Kitemaker’s integration with Sentry allows you to automatically connect issues in Sentry with work items in Kitemaker."
---

> 
> This integration is maintained and supported by the company that created it. For more details, or questions, please feel free to reach out to hi@kitemaker.co!
> ## Install and Configure

Reviewer: ```suggestion

This integration is maintained and supported by the company that created it. Feel free to reach out to hi@kitemaker.co if you need additional information or have any questions.

## Install and Configure
```

<Note>

Sentry owner, manager, or admin permissions are required to install this integration.

</Note>

> 1. Navigate to **Settings > Integrations > Kitemaker**

Reviewer: ```suggestion
1. Go to [Sentry](https://sentry.io/) and click on **Settings > Integrations > Kitemaker**
```
